### PR TITLE
Feature: Run migrations using docker-compose

### PR DESCRIPTION
**Pull request by hoh**

### CODE REVIEW

File: docs/guides/upgrade.rst [hunk 1]
```diff
@@ -41,17 +41,17 @@ Upgrade your node
...
+    docker-compose [-f <docker-compose-file>] down
+
+    docker-compose [-f <docker-compose-file>] \
+            run pyaleph /opt/pyaleph/deployment/migrations/config_updater.py \
             --key-dir /opt/pyaleph/keys \
             --key-file /opt/pyaleph/node-secret.key \
             --config /opt/pyaleph/config.yml \
             upgrade
 
+    docker-compose [-f <docker-compose-file>] down
```
Comment (odesenfans) [inline docs/guides/upgrade.rst:53]: down -> up -d

File: docs/guides/upgrade.rst [hunk 2]
```diff
@@ -41,17 +41,17 @@ Upgrade your node
...
-    docker run --rm -ti \
-        -v $(pwd)/keys:/opt/pyaleph/keys \
-        -v $(pwd)/node-secret.key:/opt/pyaleph/node-secret.key:ro \
-        -v $(pwd)/config.yml:/opt/pyaleph/config.yml:ro \
-        alephim/pyaleph-node:beta \
-        python3 /opt/pyaleph/migrations/config_updater.py \
+    docker-compose [-f <docker-compose-file>] down
+
+    docker-compose [-f <docker-compose-file>] \
+            run pyaleph /opt/pyaleph/deployment/migrations/config_updater.py \
```
Comment (odesenfans) [inline docs/guides/upgrade.rst:47]: Incorrect path, the correct path is `/opt/pyaleph/migrations/config_updater.py`.

**Review by odesenfans — CHANGES_REQUESTED** [review]

File: docs/guides/upgrade.rst (hunk 2 above)
Comment (hoh) [reply docs/guides/upgrade.rst:47]: The file was missing from the container.
I updated the commit to add it.

File: docs/guides/upgrade.rst (hunk 1 above)
Comment (hoh) [reply docs/guides/upgrade.rst:53]: The docker-compose `run` command will start the other services.
Do do the migration independently from restarting the service, I here suggest to first tear everything down before restarting it.

Comment (odesenfans) [reply docs/guides/upgrade.rst:53]: Okay, if you prefer.

File: docs/guides/upgrade.rst (hunk 2 above)
Comment (odesenfans) [reply docs/guides/upgrade.rst:47]: It's not missing, it's just not copied in the same place.

File: deployment/docker-build/pyaleph.dockerfile [hunk 3]
```diff
@@ -70,6 +71,7 @@ RUN rm /opt/build/extract_requirements.py /opt/build/requirements.txt
 COPY deployment/migrations /opt/pyaleph/migrations
```
Comment (odesenfans) [inline deployment/docker-build/pyaleph.dockerfile:71]: @hoh see here, this is where the config updater is copied.

**Pull request merged**